### PR TITLE
Added dark theme data.

### PR DESCRIPTION
**Pull request by ritvij14**
## Related issues
Solves task 1 of #36 

## Description
I have added dark theme data including the color codes and the dark logo which I took from the Figma file. Using this data we can set colors for various widgets.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -35,6 +35,9 @@ dependencies:
   pull_to_refresh: ^1.6.1
   feather_icons_flutter: ^4.7.4
   shared_preferences: ^0.5.12
+  flutter_bloc: ^6.1.0
+  equatable: ^1.2.5
+  meta: ^1.2.3
```
Comment (sanket143) [inline pubspec.yaml]: I think these packages aren't being used in the app.

**Review by sanket143 — CHANGES_REQUESTED** [review]

File: pubspec.yaml (hunk 1 above)
Comment (ritvij14) [reply pubspec.yaml]: Oh yes, I forgot to remove them while trying the bloc pattern. I have removed them for now.

File: pubspec.lock [hunk 2]
```diff
@@ -29,6 +29,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "0.15.1"
+  bloc:
+    dependency: transitive
+    description:
+      name: bloc
+      url: "https://pub.dartlang.org"
+    source: hosted
+    version: "6.1.0"
```
Comment (sanket143) [inline pubspec.lock]: run `flutter pub get` once to update `pubspec.lock` too.

**Review by sanket143 — CHANGES_REQUESTED** [review]

File: pubspec.lock (hunk 2 above)
Comment (ritvij14) [reply pubspec.lock]: Done.

**Review by sanket143 — APPROVED** [review]

**Review by janvig — APPROVED** [review]

**Comment by sanket143** [comment]
give @ritvij14 75 points

**Pull request merged**